test/core/end2end/end2end_tests.cc

```diff
@@ -56,6 +56,7 @@ Slice RandomBinarySlice(size_t length) {
   std::vector<uint8_t> output;
   output.resize(length);
   for (i = 0; i < length; ++i) {
+
```

Comment:
remove this?

Reply:
Ugh, yep, I didn't see your comment. Followup PR here https://github.com/grpc/grpc/pull/37988.